### DISCUSSION TITLE
Translation of documents for building mmdeploy 翻译ort/trt后端编译中文文档

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#553] [MMDeploy]
# Translation of documents for building mmdeploy 
 翻译ort/trt后端编译中文文档 

Please translate the following documents from English into Chinese:
1. [ONNXRuntime Backend]([https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/05-supported-backends/onnxruntime.md)](https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/05-supported-backends/onnxruntime.md)) 
2. [TensorRT Backend]([https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/05-supported-backends/tensorrt.md)](https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/05-supported-backends/tensorrt.md)) 

将以下英文文档翻译为中文
[https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/05-supported-backends/onnxruntime.md](https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/05-supported-backends/onnxruntime.md) 
[https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/05-supported-backends/tensorrt.md](https://github.com/open-mmlab/mmdeploy/blob/main/docs/zh_cn/05-supported-backends/tensorrt.md) 

## Task Information

**Repository**：MMDeploy

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：blue-q

**Claim Time**：2023年07月25日

**PR Link**：https://github.com/open-mmlab/mmdeploy/pull/2320

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by blue-q** [comment]
https://github.com/open-mmlab/mmdeploy/pull/2320
